packages/node_modules/pouchdb-adapter-indexeddb/src/setup.js

```diff
@@ -135,7 +135,7 @@ function openDatabase(openDatabases, api, opts, resolve, reject) {
     };
 
     idb.onversionchange = function () {
-      console.log('Database was made stale by being opened in another tab');
+      console.log('Database was made stale, closing handle');
```

Comment:
I also made this error a little more generic since it can happen for lots of reasons other than opening another tab